### PR TITLE
Blog post about amqplib-easy

### PR DESCRIPTION
**Pull request by apechimp**

### CODE REVIEW

**Comment by apechimp** [comment]
@lanetix/engineers review at your leisure.

File: _data/authors/mikeatkins.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name: 'Mike Atkins'
```
Comment (DannyDouglass) [inline _data/authors/mikeatkins.yml]: Blog post by @apechimp!!!

**Comment by DannyDouglass** [comment]
Looks good to me - :rocket: :beetle: away! Way to step up and get that first blog post out there. :100: :dizzy:

**Comment by OronNadiv** [comment]
Great job @apechimp :+1: :+1: :+1:

**Pull request merged**